src/main.py

```diff
@@ -167,36 +171,153 @@ async def chat(interaction: Interaction, query: str | None, file: Attachment | N
         else:
+            self.currentpage = page
+        
+        # Disable buttons in invalid states (using or does not work)
+        for item in self.children:
+            if isinstance(item, ui.Button):
+                if item.label == "Next":
+                    item.disabled = self.currentpage >= len(self.commands)-1
+                if item.label == "End":
+                    item.disabled = self.currentpage >= len(self.commands)-1
+                if item.label == "Back":
+                    item.disabled = self.currentpage == 0
+                if item.label == "Start":
+                    item.disabled = self.currentpage == 0
+                
+        # Define embed
+        embed = Embed(
+                title="Invalid embed",
+                description="Something went wrong!",
+                color=Color.yellow(),
+            )
+        
+        if self.currentpage < 0: # Log invalid case for current page
+            logging.warning(
+                f"Help Menu: Invalid Help page of value {self.currentpage} was created."
+            )
+
+        if self.currentpage == 0: # Main help menu for displaying different groups with limited detail
+            embed = Embed( 
+                title="DuckBot",
+                description="DuckBot is the CS Club's Discord bot, created by the CS Club Open Source Team.",
+                color=Color.yellow(),
+            )
+            # Adding group names
+            for command in self.commands:
+                if isinstance(command, app_commands.Group):
+                    embed.add_field(
+                        name=f"/{command.name}", value=f"{command.description}", inline=False
+                    )
+            # Misc command
             embed.add_field(
-                name=f"/{command.name}", value=command.description, inline=False
+                name="Misc", value="Miscellaneous commands:", inline=False
             )
-    await interaction.response.send_message(embed=embed)
+            for command in self.commands:
+                if not isinstance(command, app_commands.Group):
+                    embed.add_field(
+                        name=f"/{command.name}", value=f"{command.description}", inline=False
+                    )
+        
+        elif self.currentpage > 0: # Menu for differnt groups
+            command = self.commands[self.currentpage]
+            # Adding fields for groups
+            if isinstance(command, app_commands.Group):
+                embed = Embed(
+                    title=f"{command.name} commands",
+                    description=f"{command.description}",
+                    color=Color.yellow(),
+                )
+                for subcommand in command.commands:
+                    embed.add_field(
+                        name=f"/{command.name} {subcommand.name}",
+                        value=subcommand.description,
+                        inline=True,
+                    )
```

Comment:
For skullboard 
![image](https://github.com/user-attachments/assets/3a355aa0-5843-4f94-82c0-ae897fddda00) Should this change be disregarded or only change the group names to be affected?

Reply:
> For skullboard ![image](https://private-user-images.githubusercontent.com/161260651/378405992-3a355aa0-5843-4f94-82c0-ae897fddda00.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk1MTEzNDksIm5iZiI6MTcyOTUxMTA0OSwicGF0aCI6Ii8xNjEyNjA2NTEvMzc4NDA1OTkyLTNhMzU1YWEwLTU4NDMtNGY5NC04MmMwLWFlODk3ZmRkZGEwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMVQxMTQ0MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTg5YTAyMWI0ODdkM2M0ZmE1NmFjMTY4MGFmZTkwMzY3YTJkNWQ1NmNiMzkzNmRiODQ2MWUwOTEwZWQzY2VhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.82HUbPNiwd0nvE5w7obSyMHNMLEJzogxHnZbT-xYX88) Should this change be disregarded or only change the group names to be affected?

Nevermind this issue was caused by the function name not matching command title. The question about only changing the group names still stands.

Reply:
In that case keep it lowercase in the command itself and have a helper function to convert it to title case for the help menu (doing `Faq Commands` should be fine if you can't do `FAQ Commands`).

Reply:
Would having the command listed differently from their actual slash command not confuse users?

Reply:
I have added a function to capitalize commands on the first page as shown:
![image](https://github.com/user-attachments/assets/6ea8c17c-3ddf-4f7b-8fc9-ac88f2522e66)
Then the heading for each page is capitalized as shown:
![image](https://github.com/user-attachments/assets/0f7bd055-07b3-4f73-8198-b5b7de7afda1)
This preserves the command to how it can be called while capitalizing the headings.